bats/tests/profile/invalid-locked-k8s-version.bats

```diff
@@ -0,0 +1,52 @@
+load '../helpers/load'
+
+local_setup() {
+    RD_USE_PROFILE=true
+}
+
+@test 'initial factory reset' {
+    factory_reset
+}
+
+@test 'create profile' {
+    PROFILE_TYPE=$PROFILE_LOCKED
+    RD_USE_PROFILE=true
+    create_profile
+    profile_exists
```

Comment:
This is not needed here, as we are testing the app, not the helper functions.

We will create unit tests for the `tests/helpers/profile.bash` functions in the future to check this kind of relationship.

Reply:
Yeah, obv :)